pennylane/wires.py

```diff
@@ -81,37 +68,33 @@ class Wires(Sequence):
     """
 
     def __init__(self, wires):
-        self._labels = _process(wires)
-
-    def __getitem__(self, idx):
-        """Method to support indexing. Returns a Wires object representing a single wire."""
-        return Wires(self.labels[idx])
-
-    def __len__(self):
-        """Method to support ``len()``."""
-        return len(self.labels)
+        """ Extend the OrderedSet.__init__ by allowing for structured inputs """
+        wires_tuple = _process(wires)
+        super().__init__(wires_tuple)
+
+    def __getitem__(self, key):
+        _super = super()
```

Comment:
Is there a performance increase by only querying `super()` once?

Reply:
No, because `_super` eventually will only be referred to "once", meaning in one `return` statement. Actually this is _more_ expensive because `super()` is stored as `_super`. But this would be optimization on the level of memory allocation, I think...
I did it this way because calling `super()` within the generator for the `isinstance(key, slice)` case does not work and like this the two cases are handled more homogeneously. Same for `__contains__` and `indices`.